docker/pyproject.toml

```diff
@@ -14,6 +14,7 @@ Cerberus = "^1.3.4"
 glom = "^20.11.0"
 python-dotenv = "^0.18.0"
 GitPython = "^3.1.18"
+"backports.cached-property" = "^1.0.1"
```

Comment:
cc @bojiang - `./docker` is a separate python project, make sure to update its dependency when making changes